lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
Or you could just monkeypatch the `primary_stylesheet_data` method. Replacing the whole file is going to be much harder to maintain.